llvm/test/Transforms/LoopIdiom/processloopstoreforloopload-drop-debugloc.ll

```diff
@@ -0,0 +1,100 @@
+; RUN: opt < %s -passes=loop-idiom -S | FileCheck %s
+
+; Function Attrs: noinline nounwind uwtable
+define dso_local void @fun(ptr noalias noundef %a, ptr noalias noundef %b) #0 !dbg !10 {
+
+; CHECK-LABEL: entry:
+; CHECK: call void @llvm.memcpy.p0.p0.i64{{\(.*\)$}}
```

Comment:
Same here, can you comment what is being tested?

Reply:
Thanks, I add the comment to explain why and what it checks.